hs/src/Reach/Debugger.hs

```diff
@@ -1,4 +1,5 @@
 {-# OPTIONS_GHC -Wno-missing-export-lists #-}
+{-# OPTIONS_GHC -fno-warn-incomplete-patterns #-}
```

Comment:
Heresy!

Reply:
@jeapostrophe without this i basically get a billion useless warnings from the compiler: 

```
Patterns not matched:
            (ADD,
             ((V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.S# _)):
              (V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.S# _)):_:_))
            (ADD,
             ((V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.S# _)):
              (V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.Jp# _)):_:_))
            (ADD,
             ((V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.S# _)):
              (V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.Jn# _)):_:_))
            (ADD,
             ((V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.Jp# _)):
              (V_UInt (integer-gmp-1.0.3.0:GHC.Integer.Type.S# _)):_:_))
            ...
```

Reply:
Sounds like you need `_ -> impossible "bad!"` in that function